### ISSUE TITLE
Built-in "cp" should support --dereference/-L & --force/-f flags

### ISSUE THREAD

**Issue by helmesjo** [build2/build2#233]
> -L, --dereference
    always follow symbolic links in SOURCE
> -f, --force                  
    if an existing destination file cannot be
    opened, remove it and try again (this option
    is ignored when the -n option is also used)

Admittedly, the built-in commands could support a lot of things, but some arguably are used more frequently than others. Copying a file (which happens to be a symlink but you actually want to copy the referenced file) seems like a common enough scenario so I think it's a good candidate for addition.
"Force" is another common one, where instead of "manually" having to make sure the destination doesn't already exists (eg. in ad hoc install recipes) the `-f` flag is a very convenient additions.

**Comment by boris-kolpackov** [comment]
BTW, it appears `-L` is currently the default. Not sure if it's a good idea though.

**Comment by helmesjo** [comment]
@boris-kolpackov Yeah, I think it's best to fix that as early as possible rather than leaving as is. That is, use defaults as expected from Unix `cp`.
